ci/legacy/.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
 
 before_script:
   - ./configure-mac_os_x.sh
+  - sudo chmod a+rw /dev/bpf*
```

Comment:
Is there a need for travis? Maybe we should remove, no?

Reply:
@seladb what do you think ?

Reply:
I didn't remember we still keep the TravisCI file. Sure, we can remove it

Reply:
done